### ISSUE TITLE
route

### ISSUE THREAD

**Issue by tundeaji1232** [fac-13/fotos#43]
a failed registration redirect me to http://localhost:3000/profile/... any reason for that?

**Comment by tundeaji1232** [comment]
Also i think you need to look through how you link your routes....get a flow to it.

**Comment by iPhatty** [comment]
Right, registration leads to a 404, but it does register my account! 👍
